### PR TITLE
planner: binding from history GA user doc

### PR DESCRIPTION
**Pull request by fzzf678**
### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
update user document for the v6.6 feature "binding from history GA"

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):
- https://github.com/pingcap/tidb/issues/39199

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by fzzf678** [comment]
@qw4990 @Reminiscent

File: sql-plan-management.md [hunk 1]
```diff
@@ -313,6 +310,16 @@ SET BINDING [ENABLED | DISABLED] FOR BindableStmt;
...
 
+#### 根据 `sql_digest` 变更绑定状态
+
+除了可以根据 SQL 语句变更对应的绑定状态以外，也可以根据 `sql_digest` 变更绑定状态：
+
+```sql
+SET BINDING [ENABLED | DISABLED] FOR SQL DIGEST 'sql_digest';
+```
+
+使用 `sql_digest` 所能变更的绑定状态和生效情况与[根据 SQL 语句变更绑定状态](#根据-sql-语句变更绑定状态)均相同，如果没有可以改变状态的绑定，则会输出一条内容为 `can't find any binding for 'sql_digest'` 的警告。
```
Comment (TomShawn) [inline sql-plan-management.md:321]:
```suggestion
使用 `sql_digest` 所能变更的绑定状态和生效情况与[根据 SQL 语句变更绑定状态](#根据-sql-语句变更绑定状态)相同。如果没有可以改变状态的绑定，则会输出一条内容为 `can't find any binding for 'sql_digest'` 的警告。
```

**Review by qw4990 — COMMENTED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM
/status LGT2

**Review by ran-huang — APPROVED** [review]
LGTM

**Pull request merged**